api/clan.go

```diff
@@ -18,6 +18,44 @@ import (
 	"github.com/uber-go/zap"
 )
 
+func logClanOwnerID(app *App, c echo.Context, gameID, clanPublicID, when, operation string) {
+	l := app.Logger.With(
+		zap.String("source", "clanHandler"),
+		zap.String("operation", operation),
+		zap.String("gameID", gameID),
+		zap.String("clanPublicID", clanPublicID),
+		zap.String("when", when),
+	)
+
+	db, err := app.GetCtxDB(c)
+	if err != nil {
+		log.E(l, "Failed to fetch db when logging clan ownerID.", func(cm log.CM) {
+			cm.Write(zap.Error(err))
+		})
+		return
+	}
+
+	clan, err := models.GetClanByPublicID(db, gameID, clanPublicID)
+	if err != nil {
+		log.E(l, "Failed to fetch clan when logging clan ownerID.", func(cm log.CM) {
+			cm.Write(zap.Error(err))
+		})
+		return
+	}
+	if clan == nil {
+		log.E(l, "Clan do not exists when logging clan ownerID.", func(cm log.CM) {
+			cm.Write(zap.Error(err))
+		})
+		return
+	}
+
+	log.I(l, "Logged clan ownerID successfully.", func(cm log.CM) {
+		cm.Write(
+			zap.Int64("previousOwnerPublicID", clan.OwnerID),
```

Comment:
Isn't "ownerID" a more accurate name? The ID we're logging isn't a public id and also, it is the current owner and not the previous one.

For the message I'd use a simpler string like "change clan owner"